server/database/config/connection.js

```diff
@@ -0,0 +1,25 @@
+const { Pool } = require("pg");
+const { parse } = require("url");
+
+let dbURL = process.env.DATABASE_URL;
+
+if (process.env.NODE_ENV === "test") dbURL = process.env.TEST_DATABASE_URL;
```

Comment:
You forget to add conditions for the `development` and `production` environments.
I think it would look nicer if you use switch. Just a thought! It is up to you :smile: 

Also, edit scripts in package json

Reply:
development and production environment will be the same ain't them?